libs/wingc/src/lsp/completions.rs

```diff
@@ -856,11 +873,64 @@ let j = MutJson {};
 	);
 
 	test_completion_list!(
-		renamed_builtin_type_statics,
+		multiple_dots_no_completions,
 		r#"
-str.
-  //^
+let x = Json 1;
+x..
+ //^
+"#,
+		assert!(multiple_dots_no_completions.is_empty())
+	);
+
+	test_completion_list!(
+		inside_class_declaration_empty,
+		r#"
+class Foo {
+   
+//^
+}
+"#,
+		assert!(inside_class_declaration_empty.is_empty())
+	);
```

Comment:
enhancement idea: We could give a hint for writing an init function. Also any accessor that's a valid to start a member/method declaration could be valid here (`inflight`, `static`, `pub`, etc.), though I'm not sure if other languages provide these kinds of completions in this spot or not

Reply:
Totally, I think there is room for a few of them. I think most languages have them here.

I'll add the snippet for inflight here, but I'll hold off on other keywords for now since there is likely a more holistic change there (e.g. given a position, what keywords are valid)